FoursquareTop/Constants/Images.swift

```diff
@@ -8,6 +8,8 @@
   typealias Image = NSImage
 #endif
 
+// swiftlint:disable file_length
```

Comment:
Fixed in the other PR. 

It seemed to be a new "feature" with the swiftgen I'm running.